src/main/java/seedu/duke/parser/CommandParser.java

```diff
@@ -52,10 +52,6 @@ public static Command parse(String fullCommandInput) throws MoolahException {
         // Parses the command word from user input
         command = getCommand(commandWordInput, parametersInput);
 
-        if (command instanceof FindCommand) {
-            return command;
-        }
-
         // Parses the parameters from user input to set up the parameters for the command
```

Comment:
Great job in standardising the find command to the same structure used by other commands! Now our code looks much cleaner, and the user guide is also more straightforward now!